automation/src/main/java/org/greenplum/pxf/automation/structures/tables/utils/TableFactory.java

```diff
@@ -655,6 +656,29 @@ public static ExternalTable getPxfJdbcWritableTable(String name, String[] fields
         return getPxfJdbcWritableTable(name, fields, dataSourcePath, null, null, null, customParameter);
     }
 
+    /**
+     * Generate a PXF External Writable or Foreign Table using JDBC profile.
+     *
```

Comment:
From code we are returning a readable external table, so should it be something like the following?
```suggestion
     * Generate a PXF External Readable or Foreign Table using JDBC profile.
```